tests/test_butler.py

```diff
@@ -976,6 +976,9 @@ def testPruneCollections(self):
         # dataset type
         butler.registry.removeDatasetType(datasetType.name)
 
+        with self.assertLogs("lsst.daf.butler.registries.sql", "INFO"):
```

Comment:
We deleted this entire test in #775 so we probably need to move this code and the 3 lines above to the end of the `testRemoveRuns()` method.